code/modules/unit_tests/say.dm

```diff
@@ -112,20 +104,54 @@
 	var/mob/living/carbon/human/speaker = allocate(/mob/living/carbon/human)
 	var/mob/living/carbon/human/listener = allocate(/mob/living/carbon/human)
 
-	RegisterSignal(speaker, COMSIG_GLOB_LIVING_SAY_SPECIAL, .proc/handle_raw_speech)
 	RegisterSignal(speaker, COMSIG_MOB_SAY, .proc/handle_speech)
 	RegisterSignal(listener, COMSIG_MOVABLE_HEAR, .proc/handle_hearing)
 
 	speaker.forceMove(run_loc_floor_bottom_left)
 	// move listener 1 tiles away
 	listener.forceMove(locate(run_loc_floor_bottom_left.x + 1, run_loc_floor_bottom_left.y, run_loc_floor_bottom_left.z))
-	speaker.say("The quick brown fox jumps over the lazy dog", spans = list(SPAN_YELL))
 
-	TEST_ASSERT(handle_raw_speech_result, "Handle raw speech signal was not fired")
+	// speaking
+	speaker.say("The quick brown fox jumps over the lazy dog")
 	TEST_ASSERT(handle_speech_result, "Handle speech signal was not fired")
 	TEST_ASSERT(handle_hearing_result, "Handle hearing signal was not fired")
+	handle_speech_result = FALSE
+	handle_hearing_result = FALSE
```

Comment:
Instead of doing this, just make these a counter that ticks up, and check the times it was called.

Reply:
I didn't like how I was c/p'ing the same code repeatedly so I refactored it into a proc that resets those vars properly.  Look at https://github.com/tgstation/tgstation/pull/69799/commits/970a637fe63743b01e405073e1be9581fa2be20c to see how ugly it was looking.  I think it's way more readable now and allows for more testing in the future.